src/design-system/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import Badge from './Badge';
+
+export default {
+  title: 'Badge',
+  component: Badge,
+} as ComponentMeta<typeof Badge>;
+
+const Template: ComponentStory<typeof Badge> = (args) => <Badge {...args} />;
+
+export const Primary = Template.bind({});
+
+Primary.args = {
+  text: 'Primary badge',
```

Comment:
```suggestion
  children: 'Primary badge',
```

I think this got missed with the switch from `text` to accepting `children` 

Without it, the story won't render any text inside the badge